src/App.js

```diff
@@ -1,16 +1,47 @@
 import React, { Component } from "react";
-
+import Heading from "./Heading";
 import Bookings from "./Bookings";
 import Restaurant from "./Restaurant";
+import TouristInfoCards from "./TouristInfoCards";
+import SearchResults from "./SearchResults";
+import Footer from "./Footer";
 import "./App.css";
 
 class App extends Component {
   render() {
     return (
       <div className="App">
-        <header className="App-header">CYF Hotel</header>
+        <Heading />
         <Bookings />
+        <TouristInfoCards
```

Comment:
This component name implies that it shows multiple cards but actually it shows just 1. That's not really a big problem but it would be nice to fix.

I would rename your existing component to `TouristInfoCard` (no "s" at the end) and then create a new component called `TouristInfoCards` which renders all 3 of the cards.